### PR TITLE
Update to better reflect current state

### PR DESCRIPTION
**Pull request by helhum**

### CODE REVIEW

File: Documentation/MigrateToComposer/MigrationSteps.rst [hunk 1]
```diff
@@ -56,20 +51,20 @@ require`. The full syntax is::
 
 **Example**::
 
-    composer require typo3/cms:~7.6.0
+    composer require typo3/cms:~9.5.0
```
Comment (mbrodala) [inline Documentation/MigrateToComposer/MigrationSteps.rst]: This is impossible, here we must also use `typo3/minimal`. Also: why not `^9.5` or `^9`?

File: Documentation/MigrateToComposer/MigrationSteps.rst [hunk 2]
```diff
@@ -35,16 +35,11 @@ your web root. At the moment, only these few lines are required:
...
-            "typo3/cms": {
-                "cms-package-dir": "{$vendor-dir}/typo3/cms",
-                "web-dir": "public"
-            }
-        }
+        ]
     }
 
-You must set the correct name of your web root folder in property ``web-dir``.
+Note, that you only need to include this composer repository, if you have to use
```
Comment (mbrodala) [inline Documentation/MigrateToComposer/MigrationSteps.rst]: I'd love to see properly cased "Composer" everywhere. ;-)

File: Documentation/MigrateToComposer/MigrationSteps.rst [hunk 3]
```diff
@@ -85,29 +80,30 @@ As already written above, the line to install TYPO3 7 LTS would be::
...
-While installing TYPO3 8 LTS works with this line::
+While installing TYPO3 9 LTS works with this line::
 
-    composer require typo3/cms:~8.7.0
+    composer require typo3/minimal:~9.5.0
 
 .. _composer-migration-require-subtree-packages:
 
-The New Way: Add Only Code, You Need
+Add Only Code, You Need
```
Comment (mbrodala) [inline Documentation/MigrateToComposer/MigrationSteps.rst]: We should drop the comma here.

Comment (helhum) [reply Documentation/MigrateToComposer/MigrationSteps.rst]: done

File: Documentation/MigrateToComposer/MigrationSteps.rst (hunk 2 above)
Comment (helhum) [reply Documentation/MigrateToComposer/MigrationSteps.rst]: done

File: Documentation/MigrateToComposer/MigrationSteps.rst (hunk 1 above)
Comment (helhum) [reply Documentation/MigrateToComposer/MigrationSteps.rst]: done

**Pull request merged**